### ISSUE TITLE
Feature Request: Include URL option for Classic background and Images in General

### ISSUE THREAD

**Issue by BradMK** [pojome/elementor#1786]
It would be useful to be able to define the background image url in the “Classic Background Type” section, as well as any other element that has an Image option.

Use Case:
1. It would be useful if one could use images from another website without adding it the media library so if the image changes in the future it does not have to be manually updated.
2. This will also allow to use shortcodes in the URL for example if you have a Global Element on a number of pages and would like it to have a different background or any other image depending on the page title, one could make a custom shortcode in functions.php that fetches the title of the current page. Then use that shortcode in the image url such as "www.example.com/[pagetitle]-background-image.png". This will then fetch a different image depending on the page title, as long as you upload and appropriately name your images. So on a page titled "London" your image url would become "www.example.com/London-background-image.png", and on a page titled "Berlin" it would become "www.example.com/Berlin-background-image.png".
This will allow you to have a single Global Element that can be edited only once and propagated to hundreds of pages and yet have different backgrounds/images within that element on each page.

**Comment by arielk** [comment]
I merged this issue with another issue #413 in the same subject so we could deal with it more efficiently. So please let's continue the conversation there.

Thanks for your feedback!
